prime-router/src/main/kotlin/Report.kt

```diff
@@ -704,10 +712,9 @@ class Report : Logging {
         /**
          * Use a tablesaw Selection bitmap to create a mapping from this report items to newReport items.
          * Note: A tablesaw Selection is just an array of the row indexes in the oldReport that meet the filter criteria
-         * That is, selection[childRowNum] is the parentRowNum
```

Comment:
Why remove this - Is this comment wrong?

Reply:
Yes, it is referencing an argument that no longer exists, hence a lint warning